### PR TITLE
MET-5757 Integrated new Media Processor

### PR DESCRIPTION
**Pull request by JoanaCMS**

### CODE REVIEW

File: metis-media-service/src/test/java/eu/europeana/metis/mediaprocessing/extraction/MediaExtractorImplTest.java
Comment (jeortizquan) [inline metis-media-service/src/test/java/eu/europeana/metis/mediaprocessing/extraction/MediaExtractorImplTest.java:1]: update unit test for `MediaType.THREE_D`
```
@Test
  void testChooseMediaProcessor() {
    assertSame(imageProcessor, mediaExtractor.chooseMediaProcessor(MediaType.IMAGE));
    assertSame(audioVideoProcessor, mediaExtractor.chooseMediaProcessor(MediaType.AUDIO));
    assertSame(audioVideoProcessor, mediaExtractor.chooseMediaProcessor(MediaType.VIDEO));
    assertSame(textProcessor, mediaExtractor.chooseMediaProcessor(MediaType.TEXT));
    assertNull(mediaExtractor.chooseMediaProcessor(MediaType.OTHER));
  }
```

Comment (jeortizquan) [reply metis-media-service/src/test/java/eu/europeana/metis/mediaprocessing/extraction/MediaExtractorImplTest.java:1]: update unit test for `MediaType.THREE_D` to cover Media3d should not download the full file.
```
  @Test
  void testShouldDownloadForFullProcessing() {
    doReturn(true).when(imageProcessor).downloadResourceForFullProcessing();
    doReturn(true).when(textProcessor).downloadResourceForFullProcessing();
    doReturn(false).when(audioVideoProcessor).downloadResourceForFullProcessing();
    assertTrue(mediaExtractor.shouldDownloadForFullProcessing("image/unknown_type"));
    assertTrue(mediaExtractor.shouldDownloadForFullProcessing("text/unknown_type"));
    assertFalse(mediaExtractor.shouldDownloadForFullProcessing("audio/unknown_type"));
    assertFalse(mediaExtractor.shouldDownloadForFullProcessing("video/unknown_type"));
    assertFalse(mediaExtractor.shouldDownloadForFullProcessing("unknown_type"));
  }
```

File: metis-media-service/src/main/java/eu/europeana/metis/mediaprocessing/model/Media3dResourceMetadata.java
Comment (jeortizquan) [inline metis-media-service/src/main/java/eu/europeana/metis/mediaprocessing/model/Media3dResourceMetadata.java:1]: remove unused imports

**Review by jeortizquan — CHANGES_REQUESTED** [review]

File: metis-media-service/src/main/java/eu/europeana/metis/mediaprocessing/model/Media3dResourceMetadata.java
Comment (JoanaCMS) [reply metis-media-service/src/main/java/eu/europeana/metis/mediaprocessing/model/Media3dResourceMetadata.java:1]: Removed

File: metis-media-service/src/test/java/eu/europeana/metis/mediaprocessing/extraction/MediaExtractorImplTest.java
Comment (JoanaCMS) [reply metis-media-service/src/test/java/eu/europeana/metis/mediaprocessing/extraction/MediaExtractorImplTest.java:1]: Added

**Review by jeortizquan — APPROVED** [review]

**Pull request merged**